js/renderBot.js

```diff
@@ -166,12 +166,43 @@ function renderBot(directLineKey, botSecret) {
         }
     });
 
+    // calling session storage data to other pages on Bot 
+    $(function storeddata() {
+        if (sessionStorage["myKey"] != null) {
```

Comment:
keep constant on left hand side of comparison

Reply:
Resolved comments accordindly